### DISCUSSION TITLE
Error while running

### DISCUSSION THREAD

**Discussion by shivam224** [nicholaskajoh/Vehicle-Counting#57] [Q&A]
Hi,

I received following error while running the progrem.
Please find the attached screenshot.
Thanks
Shivam Agarwal
![error](https://user-images.githubusercontent.com/29938834/110432729-80123b80-80d5-11eb-8543-cb96b4fae6ab.png)

**Comment by nicholaskajoh** [comment]
Hello @shivam224, can you share the contents of your .env file?

**Reply by shivam224** [reply]
VIDEO="./data/videos/sample_traffic_scene.mp4"
DROI=[(750, 405), (1094, 398), (1569, 1028), (501, 1028)]
USE_DROI=True
SHOW_DROI=True
SHOW_COUNTS=True
MCDF=2
MCTF=3
DI=10
DETECTOR="yolo"
TRACKER="kcf"
RECORD=False
OUTPUT_VIDEO_PATH="./data/videos/output.avi"
HEADLESS=False
COUNTING_LINES=[{'label': 'A', 'line': [(667, 713), (888, 713)]}, {'label': 'B', 'line': [(1054, 866), (1423, 868)]}]
HAAR_CASCADE_PATH="./data/detectors/haarcascade/car.xml"
TFODA_WEIGHTS_PATH="./data/detectors/tfoda/faster_rcnn_inception_v2_coco_2018_01_28.pb"
TFODA_CONFIG_PATH="./data/detectors/tfoda/faster_rcnn_inception_v2_coco_2018_01_28.pbtxt"
TFODA_MODEL_DIR="./data/detectors/tfoda/faster_rcnn_inception_v2_coco_2018_01_28/saved_model"
TFODA_CLASSES_PATH="./data/detectors/coco_classes.txt"
TFODA_CLASSES_OF_INTEREST_PATH="./data/detectors/coco_classes_of_interest.txt"
TFODA_CONFIDENCE_THRESHOLD=0.5
YOLO_WEIGHTS_PATH="./data/detectors/yolo/yolov3.weights"
YOLO_CONFIG_PATH="./data/detectors/yolo/yolov3.cfg"
YOLO_CLASSES_PATH="./data/detectors/coco_classes.txt"
YOLO_CLASSES_OF_INTEREST_PATH="./data/detectors/coco_classes_of_interest.txt"
YOLO_CONFIDENCE_THRESHOLD=0.5
DETECTRON2_CONFIDENCE_THRESHOLD=0.5
DETECTRON2_CONFIG_PATH="./data/detectors/detectron2/Base-RCNN-FPN.yaml"
DETECTRON2_WEIGHTS_PATH="./data/detectors/detectron2/R50-FPN_model_final_b275ba.pkl"
DETECTRON2_NUM_CLASSES=80
DETECTRON2_CLASSES_PATH="./data/detectors/coco_classes.txt"
DETECTRON2_CLASSES_OF_INTEREST_PATH="./data/detectors/coco_classes_of_interest.txt"
ENABLE_CONSOLE_LOGGER=True
ENABLE_FILE_LOGGER=True
ENABLE_LOGSTASH_LOGGER=False
LOG_FILES_DIRECTORY="./data/logs/"
LOG_IMAGES=False
DEBUG_WINDOW_SIZE=(858, 480)
HUD_COLOR=(255, 0, 0)

**Reply by nicholaskajoh** [reply]
@shivam224 Can you print out `_bounding_boxes` [here](https://github.com/nicholaskajoh/ivy/blob/8fdb692799adcc5ab52e9f7f7836fb6e155ab280/ObjectCounter.py#L41) and share the results? I suspect there might  be a bug/misconfig somewhere. Each bounding box is meant to be a list of 4 integers.
